cmd/build_test_matrix/main.go

```
@@ -102,6 +104,11 @@ func getGithubActionMatrixForTests(e2eRootDirectory, suite string, exlcudedItems
 			return fmt.Errorf("failed extracting test suite name and test cases: %s", err)
 		}
 
+		testName := os.Getenv(testNameEnv)
+		if testName != "" && contains(testName, testCases) {
```

Should we return an error if the test name provided doesn't exist?

---

nice catch, updated!

---

An empty testName is valid? Should the else just be `else if testName != ""`?

---

whooops lol